packages/terra-consumer-nav/src/components/nav-items/NavItem.jsx

```diff
@@ -11,30 +11,36 @@ const cx = classNames.bind(styles);
 
 const propTypes = {
   ...navItemShape,
+
   /**
    * The id of the toggle component to be toggled.
    */
   toggleId: PropTypes.number,
+
   /**
    * Function callback when the toggle component is clicked.
    */
-  handleToggle: PropTypes.func.isRequired,
+  handleToggle: PropTypes.func,
```

Comment:
handleToggle is not required ?

Reply:
There's a [default prop for it](https://github.com/cerner/terra-consumer/pull/116/files/ac4c56a6eb58fcb68df5f0b5e8715ad3b38a7418#diff-e663a6b66abb4db75904c6be88b88b38R44) and this was a lint error with having a default prop and being `isRequired`